doc/man1/openssl-verification-options.pod

```diff
@@ -438,11 +674,13 @@ L<openssl-smime(1)>,
 L<openssl-cmp(1)>,
 L<openssl-cms(1)>
 
-
 =head1 HISTORY
 
 The checks enabled by B<-x509_strict> have been extended in OpenSSL 3.0.
 
+Previous versions of OpenSSL assume certificates with matching subject
+name are identical and mishandled them.
```

Comment:
This is not true as written.  What versions?  What does "mishandle" mean?  Is this necessary to say?

Reply:
Again this is some text I just moved here (as suggested by @t8m).
Shall we drop it?

Reply:
Drop it is my vote.

Reply:
Yep, drop.